tensorflow/tools/ci_build/rel/ubuntu/cpu_arm64_pip.sh

```diff
@@ -68,18 +68,6 @@ export TF_TEST_TARGETS="${DEFAULT_BAZEL_TARGETS} \
     -//tensorflow/python/data/kernel_tests:iterator_test_cpu \
     -//tensorflow/python/data/kernel_tests:iterator_test_gpu \
```

Comment:
`iterator_test_gpu` was added by https://github.com/tensorflow/tensorflow/pull/56811. I don't think it needs to be here since GPU tests are not included in the test tags. Should we remove it?

Reply:
In the interest of time, we can do this in a later PR if it should be removed. I'll get this PR in now.